pkg/agent/types/networkpolicy.go

```diff
@@ -77,3 +77,13 @@ func (p *Priority) Less(p2 Priority) bool {
 	}
 	return p.TierPriority > p2.TierPriority
 }
+
+type RuleMetric struct {
+	Bytes, Packets, Connections uint32
```

Comment:
Could you rename Connections to Sessions to be consistent with the fields in API.

Reply:
Sure, updated.